src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/index.tsx

```diff
@@ -49,6 +49,7 @@ function CustomBottomTabNavigator({initialRouteName, children, screenOptions, ..
         <ScreenWrapper
             testID={CustomBottomTabNavigator.displayName}
             shouldShowOfflineIndicator={false}
+            shouldEnableKeyboardAvoidingView={false}
```

Comment:
```suggestion
            shouldEnableKeyboardAvoidingView={false}
            shouldEnablePickerAvoiding={false}
```

NAB. The picker avoid view is not needed either

Reply:
Added it as well! Thanks 🙏